src/libclipper/include/clipper/datatypes.hpp

```diff
@@ -20,6 +20,11 @@ enum class InputType {
   Strings = 4,
 };
 
+enum class OutputType {
```

Comment:
Decided to consolidate the `OutputType` enums and the to/from-string functions in `datatypes.hpp/cpp`. Let me know what you think.